public/assets/icons/icons.css

```
@@ -0,0 +1,56 @@
+        url('./fonts/icomoon.ttf?4etbac') format('truetype'),
+        url('./fonts/icomoon.woff?4etbac') format('woff'),
+        url('./fonts/icomoon.svg?4etbac#icomoon') format('svg');
+    font-weight: normal;
+    font-style: normal;
+    font-display: block;
+}
+
+[class^='icon-'],
+[class*=' icon-'] {
+    /* use !important to prevent issues with browser extensions that change fonts */
+    font-family: 'icomoon' !important;
```

Biome (1.9.4)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)